lib/src/middleware/helix_idxs_change.dart

```diff
@@ -12,6 +12,24 @@ helix_idxs_change_middleware(Store<AppState> store, dynamic action, NextDispatch
     Set<int> existing_idxs = store.state.design.helices.keys.toSet();
     Set<int> old_idxs = action.idx_replacements.keys.toSet();
     Set<int> remaining_idxs = existing_idxs.difference(old_idxs);
+    var new_indices = action.idx_replacements.values.toSet();
+
+    if (new_indices.length != action.idx_replacements.length) {
+      Set<int> intsSoFar = {};
+      List<int> all_helices = action.idx_replacements.values.toList();
+
+      for (var old_index in old_idxs) {
+        var new_index = action.idx_replacements[old_index];
+        if (intsSoFar.contains(new_index)) {
+          var msg = 'First duplicate found for helix ${old_index} with new index ${new_index}. ';
```

Comment:
Sorry, try it one more time:

![image](https://user-images.githubusercontent.com/19274365/183269693-3792310d-fe6d-4a45-a441-ee3bcdd4e282.png)

Imagine you're a user and there's a HUGE list of indices there. If they tried to assign index a to b and also index c to b, then they'll want to know all three of a, b, c. The current error message is only telling them two of these.

Try this error message: "You tried to assign helices 4 and 5 both to 0. Each helix must have a *unique* new index; make sure all the integers you write are distinct from each other and do not appear elsewhere in the design."